Dockerfile

```diff
@@ -19,4 +19,7 @@ RUN apk --update --no-cache add \
 RUN apk --no-cache add tzdata
 
 COPY --from=build-env /go/bin/search-service /go/bin/search-service
+RUN addgroup -S app && adduser -S -g app app \
```

Comment:
Sorry unsure what this line does

Reply:
Creates a user-group called app, then creates a user called app which is added to that group.
It's better to create an empty group with no permissions as a user has to belong to at least one group, and this gives the app user zero permissions via group inheritance